### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by justinsb**
Experimenting with some typo finding tools, and this is a batch I found in the comments.

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/cluster/v1alpha1/machine_types.go [hunk 1]
```diff
@@ -160,7 +160,7 @@ type MachineStatus struct {
 	LastOperation *LastOperation `json:"lastOperation,omitempty"`
 
 	// Phase represents the current phase of machine actuation.
-	// Eg. Pending, Running, Terminating, Failed etc.
+	// e.g. Pending, Running, Terminating, Failed etc.
```
Comment (davidewatson) [inline pkg/apis/cluster/v1alpha1/machine_types.go]: This is sentence and should be capitalized.

File: pkg/apis/cluster/v1alpha1/machine_types.go [hunk 2]
```diff
@@ -174,11 +174,11 @@ type LastOperation struct {
 	LastUpdated *metav1.Time `json:"lastUpdated,omitempty"`
 
 	// State is the current status of the last performed operation.
-	// Eg.Processing, Failed, Successful etc
+	// e.g. Processing, Failed, Successful etc
```
Comment (davidewatson) [inline pkg/apis/cluster/v1alpha1/machine_types.go]: Same as above.

File: pkg/apis/cluster/v1alpha1/machine_types.go [hunk 3]
```diff
@@ -174,11 +174,11 @@ type LastOperation struct {
...
 	State *string `json:"state,omitempty"`
 
 	// Type is the type of operation which was last performed.
-	// Eg. Create, Delete, Update etc
+	// e.g. Create, Delete, Update etc
```
Comment (davidewatson) [inline pkg/apis/cluster/v1alpha1/machine_types.go]: Same.

**Comment by alejandrox1** [comment]
@justinsb what tools are you using?

**Comment by justinsb** [comment]
Well obviously I wrote my own spell checker :-)

File: pkg/apis/cluster/v1alpha1/machine_types.go (hunk 1 above)
Comment (justinsb) [reply pkg/apis/cluster/v1alpha1/machine_types.go]: Thanks @davidewatson - should be fixed :-)

**Comment by justinsb** [comment]
It's a very basic spell checker, that really just does tokenization and then matches against static wordlists.  I'm really seeing whether we actually use a very small subset of language and thus we don't need a "real" spell checker.  Also by building the wordlist up from real-world projects I figure we can both deal with go's camel casing, the kubernetes made-up words, and avoid words we don't want to use (e.g. minion.  But for now I just though I would fix the set of typos this discovered in the comments (this is all the typos it found, though there's a per-project skiplist of about 100 words where I'm really punting on some of the trickier style points (e.g. `node-object` or `Node object`).  And kudos for @davidewatson for spotting the `e.g.` vs `E.g.` where I thought I was safe!

**Pull request merged**